langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -75,6 +79,13 @@ type XmlType xml;
 # Returns an iterator over the xml items of an xml sequence.
 #
 # # Each item is represented by an xml singleton.
+# 
+# ```ballerina
+# object {
+#     public isolated function next() returns record {|xml value;|}?;
+# } iterator = xml `<book><title>Ballerina</title></book><book><title>Java</title></book>`.iterator();
```

Comment:
Please refer to https://github.com/ballerina-platform/ballerina-lang/pull/38981#discussion_r1060286750

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/748ab266475d51db8f993ce05fca56be34858b08

Reply:
There are names of programming languages in the updated changes.
```
Java guide, Javascript Guide
```

Don't we need to get rid of these words.

wdyt? @MaryamZi @sanjana

Reply:
Yes.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39130/commits/845ad4832590ca4ff4fbcbd72745bb1e34163345